`make venv` complains that the 'requests' distribution was not found and is required

Steps to reproduce: 

1. git clone .. 
2. make venv

```
20-02-17 17:24 ➜  cachito git:(master) make venv
virtualenv --python=python3 venv && venv/bin/python setup.py develop && venv/bin/pip install -r requirements-dev.txt
created virtual environment in 300ms CPython3Posix(dest=/Users/yorgos/dev/redhat/productization/cachito/venv, clear=False, global=False) with seeder FromAppData pip=latest setuptools=latest wheel=latest app_data_dir=/Users/yorgos/Library/Application Support/virtualenv/seed-v1 via=copy
running develop
running egg_info
creating cachito.egg-info
writing cachito.egg-info/PKG-INFO
writing dependency_links to cachito.egg-info/dependency_links.txt
writing entry points to cachito.egg-info/entry_points.txt
writing requirements to cachito.egg-info/requires.txt
writing top-level names to cachito.egg-info/top_level.txt
writing manifest file 'cachito.egg-info/SOURCES.txt'
reading manifest file 'cachito.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching '*' under directory 'cachito/web/templates'
writing manifest file 'cachito.egg-info/SOURCES.txt'
running build_ext
Creating /Users/yorgos/dev/redhat/productization/cachito/venv/lib/python3.7/site-packages/cachito.egg-link (link to .)
Adding cachito 1.0 to easy-install.pth file
Installing cachito script to /Users/yorgos/dev/redhat/productization/cachito/venv/bin
Installing cachito-cleanup script to /Users/yorgos/dev/redhat/productization/cachito/venv/bin

Installed /Users/yorgos/dev/redhat/productization/cachito
Processing dependencies for cachito==1.0
Searching for semver
Reading https://pypi.org/simple/semver/
Downloading https://files.pythonhosted.org/packages/cb/48/009529f08dd0f71262a229f0a7948dd5d27eb0e0a9edc74e6e6860ff2925/semver-2.9.1-py2.py3-none-any.whl#sha256=095c3cba6d5433f21451101463b22cf831fe6996fcc8a603407fd8bea54f116b
Best match: semver 2.9.1
Processing semver-2.9.1-py2.py3-none-any.whl
Installing semver-2.9.1-py2.py3-none-any.whl to /Users/yorgos/dev/redhat/productization/cachito/venv/lib/python3.7/site-packages
Adding semver 2.9.1 to easy-install.pth file
Installing pysemver script to /Users/yorgos/dev/redhat/productization/cachito/venv/bin

Installed /Users/yorgos/dev/redhat/productization/cachito/venv/lib/python3.7/site-packages/semver-2.9.1-py3.7.egg
Searching for requests_kerberos
Reading https://pypi.org/simple/requests_kerberos/
Downloading https://files.pythonhosted.org/packages/ee/a2/866f2b9a60f75055137b9ad127033e397963b2c4769d4b5fab1c3c7e8be3/requests_kerberos-0.12.0-py2.py3-none-any.whl#sha256=5733abc0b6524815f6fc72d5c0ec9f3fb89137b852adea2a461c45931f5675e0
Best match: requests-kerberos 0.12.0
Processing requests_kerberos-0.12.0-py2.py3-none-any.whl
Installing requests_kerberos-0.12.0-py2.py3-none-any.whl to /Users/yorgos/dev/redhat/productization/cachito/venv/lib/python3.7/site-packages
Adding requests-kerberos 0.12.0 to easy-install.pth file

Installed /Users/yorgos/dev/redhat/productization/cachito/venv/lib/python3.7/site-packages/requests_kerberos-0.12.0-py3.7.egg
Searching for requests
Downloading https://files.pythonhosted.org/packages/14/61/85737ebe1e65cd4bf023d9e4610df70851bd7638e003b81a44a9b901feae/requests-kerberos-0.12.0.tar.gz#sha256=9d21f15241c53c2ad47e813138b9aee4b9acdd04b82048c4388ade15f40a52fd
Best match: requests kerberos-0.12.0
Processing requests-kerberos-0.12.0.tar.gz
Writing /var/folders/0h/28qktgvj01j9hp61hck2p7sc0000gn/T/easy_install-w551150z/requests-kerberos-0.12.0/setup.cfg
Running requests-kerberos-0.12.0/setup.py -q bdist_egg --dist-dir /var/folders/0h/28qktgvj01j9hp61hck2p7sc0000gn/T/easy_install-w551150z/requests-kerberos-0.12.0/egg-dist-tmp-hs13tnkx
zip_safe flag not set; analyzing archive contents...
removing '/Users/yorgos/dev/redhat/productization/cachito/venv/lib/python3.7/site-packages/requests_kerberos-0.12.0-py3.7.egg' (and everything under it)
Copying requests_kerberos-0.12.0-py3.7.egg to /Users/yorgos/dev/redhat/productization/cachito/venv/lib/python3.7/site-packages
requests-kerberos 0.12.0 is already the active version in easy-install.pth

Installed /Users/yorgos/dev/redhat/productization/cachito/venv/lib/python3.7/site-packages/requests_kerberos-0.12.0-py3.7.egg
error: The 'requests' distribution was not found and is required by cachito
make: *** [venv] Error 1
```

python version: 

```
20-02-17 17:24 ➜  cachito git:(master) python --version
Python 3.7.6
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
 celery
 gitpython
 kombu # A celery dependency but it's directly imported
-requests
 requests_kerberos
+requests
 semver

